### DISCUSSION TITLE
Ollama: 502 Server Error: Bad Gateway for url: http://localhost:11434/api/chat  Install without docker

### DISCUSSION THREAD

**Discussion by peuportier** [open-webui/open-webui#1508] [General]
# Bug Report

I get this message error : Ollama: 502 Server Error: Bad Gateway for URL: http://localhost:11434/api/chat 

## Description

My installation is without docker and was running very well. I close my computer and then reopen it. The problem happening

**Bug Summary:**
I have installed it on My Mac M1 - 14.4.1 Somona - open-WebUI without the use of docker. By running the normal process described in your doc. Running it very well. Can get the model from Ollama and run the request. I shut down my computer. Restart it and always now get this problem. 

**Steps to Reproduce:**
Just run "bash install.sh" from terminal
Go to localhost:8080
Login 
Then select my model and ask a request. Get this message error   

**Expected Behavior:**
Normally get the answer on the chat 

**Actual Behavior:**
Get nothing and have a popup shows Ollama: 502 Server Error: Bad Gateway for url: http://localhost:11434/api/chat

## Environment

- **Operating System:**  Mac OS Somona  14.4.1
- **Browser (if applicable):** [e.g., Firefox 124.0.2 or safari 17.4.1]

## Reproduction Details

**Confirmation:**

- [x ] I have read and followed all the instructions provided in the README.md.
- [ x] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots (if applicable):**
<img width="1440" alt="Screenshot 2024-04-11 at 23 25 12" src="https://github.com/open-webui/open-webui/assets/30169867/60373cf6-d0f4-4c57-b82f-1eb9a395abe9">

## Installation Method

```
git clone https://github.com/open-webui/open-webui.git
cd open-webui/

# Copying required .env file
cp -RPp .env.example .env

# Building Frontend Using Node
npm i
npm run build

# Serving Frontend with the Backend
cd ./backend
pip install -r requirements.txt -U
bash start.sh
```

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by peuportier** [comment]
More complete information maybe can help for this issue? 

[localhost.har 2.zip](https://github.com/open-webui/open-webui/files/14968126/localhost.har.2.zip)

Really need some help. Cause i can't get any response from models. I can use Ollama through terminal normally but through openwebui always get this message. Tell me what you'll need to solve that issue. You can solve into docker but not if the install isn't docker one? thanks for you time

**Comment by justinh-rahb** [comment]
We recommend most users to use our Docker installation method as this eliminates entire classes of unknowns that can prevent successful use of our project. We also recommend to ensure you've got a fully updated installation of Ollama, as older versions are known to not work with WebUI.

If you're still experiencing the same issue after updating Ollama, we'd prefer that you paste text logs from Docker for the backend if that's the issue, or from your browser developer console if that's relevant. This way we can review properly and provide more assistance.

**Comment by peuportier** [comment]
Hi, Thank you for your insight. I still prefer not to use the docker and wanna try to find out what is the problem. Ollama running the last version. Install as following the guidelines. Seems like maybe the problem is not from Your project but from Ollama himself who doesn't like the project manually being restarted.  I will first reinstall all from a clean new slate. If the problem continues, I will report here what the problem is. Meanwhile, if I find a solution I will share it with the community.

**Reply by doughnet** [reply]
Did you find the issue/cause? I been using open webui the same outside of docker and it been fine for 3 weeks. Then as of today it seems to have gone wrong and getting the 502 errors after a period of time.

**Comment by CSAnetGmbH** [comment]
Please try only http://localhost:11434 and if this don´t work, try http://127.0.0.1:11434 This could help.

**Comment by AROP-REPLICATION** [comment]
turn off the global proxy works for me

**Reply by zhourinatian** [reply]
Yes, please check your proxy settings first, turn it off or add an exception

**Reply by Mobius0916** [reply]
That works for me as well

**Reply by RobinElysia** [reply]
感谢👌
